cmd/nerdctl/images.go

```diff
@@ -142,16 +143,21 @@ func printImages(ctx context.Context, clicontext *cli.Context, client *container
 		}
 		repository, tag := imgutil.ParseRepoTag(img.Name)
 
+		config, err := img.Config(ctx, cs, platforms.DefaultStrict())
+		if err != nil {
+			return err
+		}
+
 		p := imagePrintable{
 			CreatedAt:    img.CreatedAt.Round(time.Second).Local().String(), // format like "2021-08-07 02:19:45 +0900 JST"
 			CreatedSince: timeSinceInHuman(img.CreatedAt),
-			ID:           img.Target.Digest.String(),
+			ID:           config.Digest.String(),
```

Comment:
Is this ID compatible to `nerdctl inspect`?

Reply:
nope is it not compatible with `inspect` . I don't see how I can escalate from `config.Digest` of the current platform to `target.Digest` . is there on the containerd side a pedefinit function that allows you to do that?